src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/sdk/models/database_account_create_update_parameters.py

```diff
@@ -0,0 +1,76 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from .resource import Resource
+
+
+class DatabaseAccountCreateUpdateParameters(Resource):
+    """Parameters to create and update DocumentDB database accounts.
+
+    Variables are only populated by the server, and will be ignored when
+    sending a request.
+
+    :ivar id: The unique resource identifier of the database account.
+    :vartype id: str
+    :ivar name: The name of the database account.
+    :vartype name: str
+    :ivar type: The type of Azure resource.
+    :vartype type: str
+    :param location: The location of the resource group to which the resource
```

Comment:
Since this is autogenerated, this might be simple a swagger documentation issue. Is location the "location of the resource" (which is usually the case, and may differ from the resource group's location) or is this truly the "resource group's location"? If so, you can't exactly change it since the resource group's location can't be changed, which makes this a senseless parameter.